src/leo_misc.erl

```diff
@@ -174,3 +175,19 @@ uuid(_) ->
 %% @private
 v4(R1, R2, R3, R4) ->
     <<R1:48, 4:4, R2:12, 2:2, R3:32, R4: 30>>.
+
+
+%% @doc Send "start-up complete" notification to systemd
+%%
+-spec(startup_notification() ->
+             ok).
+startup_notification() ->
+    case code:load_file(sd_notify) of
+        {module, sd_notify} ->
+            sd_notify:sd_notify(0, "READY=1"),
```

Comment:
s/sd_notify:sd_notify/SDNotify:sd_notify/

The above two amendments will avoid static checking the module existence so as a result, compile warning gone and xref passed without adding sd_notify dependency.